assets/js/modules/analytics-4/datastore/__fixtures__/audiences.json

```diff
@@ -22,6 +22,120 @@
         "exclusionDurationMode": null,
         "membershipDurationDays": 30,
         "name": "properties/12345/audiences/3",
+        "filterClauses": [
```

Comment:
Please can you update the `displayName` and `description` fields for the SK-created audiences to align with their expected values as per the defined [audience configuration](https://docs.google.com/document/d/1MGD5Djy6AeeZC4zBtHqS-lQEWD9jw0kf-IIWw-jLCFU/edit#heading=h.c3935058ied6)?

```json
			"displayName": "New visitors",
			"description": "People who visited the site for the first time",
```

```json
			"displayName": "Returning visitors",
			"description": "People who have visited your site at least once before",
```

Additionally, you've introduced tab-indented formatting to this file which was previously space-indented, so it now has a mixture of tabs and spaces. Please can you update it to be consistent one way or the other?

Reply:
Updated, please let me know if this looks good now, thanks!

Reply:
That's great, thanks!